js/app/src/components/Chatbot/Chatbot.test.ts

```diff
@@ -31,14 +31,14 @@ describe.skip("Chatbot", () => {
 		const bot = getAllByTestId("user")[0];
 		const user = getAllByTestId("bot")[0];
 
-		assert.equal(get_text(bot), "user message one");
```

Comment:
Not sure why, but getting the text doesn't work. Neither does trying to retrieve the `innerhtml`.  Maybe because the text is embedded within other svelte components? cc @pngwn

Reply:
See this comment / PR: https://github.com/gradio-app/gradio/pull/4527#issuecomment-1593276159

We can't really test this in node right now because we use `requestAnimationFrame` to work around a bug.

Reply:
Then just checking for: `assert.exists(bot);` should be enough for now I guess